### ISSUE TITLE
[Bug]: Stacks ($STX) add accounts is failing 

### ISSUE THREAD

**Issue by hugecapybara** [LedgerHQ/ledger-live#4399]
### Describe the bug

I have existing $STX accounts on ledger that I used with the Hiro wallet. Now when I try to add stx accounts to the native ledger live app, i get this error: 
<img width="479" alt="image" src="https://github.com/LedgerHQ/ledger-live/assets/142742848/c57c5556-6290-4c35-b7c1-1ace85e0ea12">

```
Cannot destructure property 'sender' of 't.stx_transfers[0]' as it is undefined.
```
which should correspond to this [line](https://github.com/LedgerHQ/ledger-live/blob/a4648d5c78e21989c9d5a3de7c7df80e05672eef/libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts#L68)

### Expected behavior

I should be able to add existing $STX accounts on ledger live

**Comment by jbencin** [comment]
I am seeing the same thing. It worked fine at first,  I saw the issue only *after* I made a couple of transactions to participate in delegated Stacking at [Xverse](https://pool.xverse.app/). So I think it handles simple STX send and receive transactions just fine, but may fail on more complex ones like contract calls. Specifically, it *must* be at least one of these two transactions that is causing an error for me:
- [0x27f43efcf22c5c1ffccadfa1daa3f5198b172a241eebb97af4e903f0706dd407](https://explorer.hiro.so/txid/0x27f43efcf22c5c1ffccadfa1daa3f5198b172a241eebb97af4e903f0706dd407?chain=mainnet)
- [0xe2d5cd446374833311e852aa29660d4e90dd01488e760f30fb8843226147131c](https://explorer.hiro.so/txid/0xe2d5cd446374833311e852aa29660d4e90dd01488e760f30fb8843226147131c?chain=mainnet)

**Comment by ren132710** [comment]
I just got the same error. I am new to Stacks and very recently setup Hiro Desktop / Ledger Nano Plus integration. Similar to the comment by @jbencin above, this error did not appear until AFTER I stacked my first STX.  I also tried to add an Account to Ledger Live and received same error. See attached screen shots. 
![ledger live - cannot destructure property 'sender' of 't stx_transfers 0 ' as it is undefined](https://github.com/LedgerHQ/ledger-l
![Accounts-List](https://github.com/LedgerHQ/ledger-live/assets/96791510/c49e55b7-7c8c-42d2-a978-82699b5f687d)
ive/assets/96791510/69a97b92-effc-42c0-a391-6f72f756f760)
